app/models/post.rb

```diff
@@ -0,0 +1,6 @@
+class Post < ActiveRecord::Base
+  validates :image, presence: true
+
+  has_attached_file :image, styles: { :medium => "640x" } #sizing any image 640xpixels
```

Comment:
Metrics/LineLength: Line is too long. [86/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)